### ISSUE TITLE
Batoto chapter detection fails

### ISSUE THREAD

**Issue by xtj-9182** [raulhaag/MiMangaNu#418]
@raulhaag 
It's seems like our batoto chapter detection fails. I made sure to login in again but I'm always getting an empty data string in line 155 of batoto.java.

**Comment by raulhaag** [comment]
Tested and work for me, try clear cookies ando check the page oficina source

**Comment by xtj-9182** [comment]
oh ok I'm gonna try that. 

Also I think it would be useful to have a setting to clear cookies. I was working on that anyway

**Comment by raulhaag** [comment]
adda and give me
- 1 spanish
- 1 english 
- 1 indonesian 
those 3 chapters

**Comment by xtj-9182** [comment]
maybe I have a login issue? When I try to go to batotoES I get an exception when going to any manga. This is something I remember fixing for normal batoto

**Comment by xtj-9182** [comment]
oh yeah looking at my cookies I have a commentmodpid something cookie but no memberid cookie, even though I'm logged in

**Comment by raulhaag** [comment]
i will need to add both check

**Comment by xtj-9182** [comment]
unfortunately I can't access my Android Studio right now but in a few hours I can do a PR with my new clear cookie settings. I think my batoto cookie probably expired or I deleted it and the new cookie is different.

**Comment by xtj-9182** [comment]
ok this is odd. I deleted all cookies and suddenly I get the memberid cookie again. I selectively deleted bato.to cookies before and for some reason that messed with it. So after deleting all cookies and logging in again I get 5 cookies from bato.to including the memberid cookie before I only got the comment cookie somehow...

anyway it seems to work now. I guess the best solution is to simply delete all cookies every now and then.
